docs/scoping.md

```diff
@@ -197,6 +197,20 @@ have unique model elements (no double instances).
 Examples see [tests/test_scoping/test_import_module.py](https://github.com/textX/textX/blob/master/tests/functional/test_scoping/test_import_module.py).
 
 
+### Included model retrieval
+
+When a model includes or references other model (model files), the scope 
+providers in `textx.scoping.providers` use the field `_tx_model_repository`
+of the model object to keep track of the included models.
+
+ * You can get a list of all included model objects of a model
+   and the model itself with
+   `textx.scoping.providers.get_all_models_including_attached_models`.
```

Comment:
I would prefer a shorter name for this function especially cause it has `attached` in its name while we haven't defined what `attached` means. I think it would be just enough for the name to be `get_included_models` or if more precision is needed (although I still prefer shorter names :) ) the name could be `get_model_with_included_models`.

Reply:
`get_included_models` could be a method on a model object. See comment bellow for `is_file_included`

Reply:
Done. As free function.